avocado/utils/genio.py

```diff
@@ -174,3 +181,21 @@ def write_one_line(filename, line):
     :type line: str
     """
     write_file(filename, line.rstrip('\n') + '\n')
+
+
+def write_file_or_fail(filename, data):
+    """
+    Write to a file and raise exception on write failure
+
+    :param filename : Path to file
```

Comment:
Oh it's everywhere. Do you use some kind of IDE which does that? I'd recommend re-configure it to sphinx default. For example Eclipse does that automatically out of the box... Anyway I'm really sorry for not spotting this earlier but could you please do this last change? Force-push is fine.